stdlib/public/core/StringUTF16View.swift

```diff
@@ -200,23 +200,6 @@ extension String {
   public init(_ utf16: UTF16View) {
     self.init(utf16._guts)
   }
-
-  #if _runtime(_ObjC)
-  // These may become less important once <rdar://problem/19255291> is addressed.
-  @available(
-    *, unavailable,
-    message: "Indexing a String's UTF16View requires a String.UTF16View.Index, which can be constructed from Int when Foundation is imported")
-  public subscript(i: Int) -> UTF16.CodeUnit {
```

Comment:
I wonder if it's possible to keep these subscripts (a message looks helpful) by adding a defaulted parameter...

Reply:
I think the situation this came up is long-gone, and these are just vestigial. The important message – that you can't subscript a `String` with an `Int` – is still there.